packages/expo-dev-launcher/ios/EXDevLauncherAppDelegate.mm

```diff
@@ -35,4 +38,12 @@ - (NSURL *)bundleURL {
   return self.bundleURLGetter();
 }
 
+- (Class)getModuleClassFromName:(const char *)name
+{
+  if (strcmp(name, "DevLoadingView") == 0) {
+    return [DevClientNoOpLoadingView class];
+  }
+  return [super getModuleClassFromName:name];
```

Comment:
Can you add a comment? It will be hard to understand why we did that in the future?